decidim-core/app/cells/decidim/card_cell.rb

```diff
@@ -32,7 +32,7 @@ def title
     end
 
     def body
-      model.try(:body) || model.try(:about) || ""
+      decidim_escape_translated(model.try(:body) || model.try(:about) || "")
```

Comment:
This isn't done in develop 
```suggestion
      translated_attribute(model.try(:body) || model.try(:about) || "")
```

Reply:
We have a different class structure in 27 vs 28 ... In 28 we have CardGCell and CardLCell, wihch in the end are using strip_tags.

Reply:
Same here